web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -45,18 +45,44 @@ export type IntegrationSpecAwsOidc = {
   roleArn: string;
 };
 
-// IntegrationStatusCode must be in sync with the text values defined
-// in the backend as these are used to determine the status color:
-// https://github.com/gravitational/teleport.e/blob/1ebe50ce2fe608dc6dd24fef205fb9caaa216a46/lib/web/ui/plugins.go#L51
-export type IntegrationStatusCode =
-  | 'Unknown'
-  | 'Running'
-  | 'Unknown error'
-  | 'Unauthorized'
-  | 'Bot not invited to channel';
+export enum IntegrationStatusCode {
+  UNKNOWN = 0,
```

Comment:
Enums values should be in UpperCamelCase https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums